README.md

```diff
@@ -118,7 +122,7 @@ You can [watch an intro to multiple-cursors at Emacs Rocks](http://emacsrocks.co
 - If you would like to keep the global bindings clean, and get custom keybindings
   when the region is active, you can try [region-bindings-mode](https://github.com/fgallina/region-bindings-mode).
 
-BTW, I highly recommend adding `mc/mark-next-like-this` to a key binding that's
+BTW, I highly recommend adding `mc/mark-next-like-this` or `mc/mark-next-like-this-word` to a key binding that's
```

Comment:
This recommendation makes most sense for only `mark-next-like-this`. The `-word` variant is basically there to avoid using expand-region first. :)